src/apps/trainings/models/network.py

```diff
@@ -128,6 +129,18 @@ def elo_uncertainty(self):
     def rating(self):
         return f"{self.elo} ± {2 * self.elo_uncertainty}"
 
+    @property
+    def download_link_html(self):
+        if self.model_file:
+            return f'<a href="{self.model_file.url}">[{download_translated}]</a>'
+        return ""
+    @property
+    def zip_download_link_html(self):
+        if self.model_zip_file:
+            return f'<a href="{self.model_zip_file.url}">[{download_translated}]</a>'
+        return ""
```

Comment:
Looks weird for me to put template in model. It breaks the single responsability principle

For me that should rather be a template tag `obj.model_file|download_link`

Reply:
You are right, thanks. I fixed it and used a template filter instead.